src/locales/cy/pages.yml

```diff
@@ -25,6 +25,12 @@ check-your-details:
   issueNumber: Issue number
   postcode: Postcode
 
+consent:
```

Comment:
Do we have welsh for these?

Reply:
Welsh translations are being obtain in separate [ticket 1368 ](https://govukverify.atlassian.net/browse/LIME-1368)